rebasehelper/application.py

```diff
@@ -172,6 +175,7 @@ def _prepare_spec_objects(self):
         self.rebase_spec_file_path = get_rebase_name(self.rebased_sources_dir, self.spec_file_path)
 
         self.spec_file = SpecFile(self.spec_file_path,
+                                  self.kwargs['changelog-entry'],
```

Comment:
I'd rather see `self.conf.changelog_entry` here. You could also pass the entire `kwargs` dict so other attributes can also be consumed by `SpecFile` class (if necessary).